bin/create-config

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env node
 // Ghost Configuration for Heroku
 
-var fs = require('fs');
-var path = require('path');
-var url = require('url');
+const fs = require('fs'),
```

Comment:
@tcyrus With respect to your code changes, I agree, all vars must die. Long live king const!

I would caution against coma-delimited assignments statements. This is mainly because they become more difficult to add, remove, and re-organize. They can also be less readable. One exception might be declaring variables at the beginning of a code block (eg let subTotal, count, current, next), but even then I find it can create maintenance challenges and coding dilemmas.
For example, consider the following:

const fun1 = (arg) => {
  let subTotal, count, current, next
  subTotal = 0
  count = 0
  //...
}
const fun2 = (arg) => {
  let current, next
  let subTotal = 0
  let count = 0
  //...
}
const fun3 = (arg) => {
  let subTotal = 0
  let count = 0
  let current
  let next
  //...
}
For myself, fun3 is the easiest to read, produce, and maintain. 

In the absence of a styleguide/linter, I've been looking at Fission's typescript-client for inspiration. I especially love the lack of semi-colons and finally decided to drop them from my own coding preferences.

Reply:
Ok. I wasn't sure about the style guidelines for this repository.

Reply:
There are none yet 😄